fofix/game/guitarscene/instruments/Drum.py

```diff
@@ -416,31 +416,34 @@ def renderFrets(self, visibility, song, controls):
                                         coord = (x,v,0), multiples = True,color = fretColor, depth = True)
 
             else:
+                #TODO finish 3d keys! keytex and friends are never defined so this block is completely undeveloped!
+
                 self.keypos = self.engine.theme.drumkeypos
                 self.keyrot = self.engine.theme.drumkeyrot
 
                 texture = None
                 model = self.keyMesh
                 if self.keytex:
-                    if n == 0:
-                        texture = self.keytexb.texture
+                    if n == 0 and self.keytexopen:
+                        texture = self.keytexopen.texture
                     elif n == 1:
-                        texture = self.keytexc.texture
+                        texture = self.keytexa.texture
                     elif n == 2:
-                        texture = self.keytexd.texture
+                        texture = self.keytexb.texture
                     elif n == 3:
-                        texture = self.keytexa.texture
-                    elif n == 4 and self.keytexopen:
-                        texture = self.keytexopen.texture
+                        texture = self.keytexc.texture
+                    elif n == 4:
+                        texture = self.keytexd.texture
 
-                if n == 4:
+                if n == 0:
                     model = self.keyMeshOpen
 
                 c = [.1 + .8 * c[0] + f, .1 + .8 * c[1] + f, .1 + .8 * c[2] + f, v]
                 self.render3DKey(texture, model, x, y, c, n, f)
 
         gl.glDisable(gl.GL_DEPTH_TEST)
 
+
```

Comment:
Remove useless line.

Reply:
What useless line? The blank line? I always double-space between methods.

Reply:
Well if we are going by pep8, Linkid is correct:
https://www.python.org/dev/peps/pep-0008/#blank-lines